### ISSUE TITLE
Have you found any other workarounds? 

### ISSUE THREAD

**Issue by kharmabum** [felix-dumit/BluetoothTest#1]
Also this workaround does not work for me on an iPhone 7 Plus running 10.3.2 beta.

**Comment by felix-dumit** [comment]
Yeah, it is still not working for me as well. Was it working for you with any devices?
I still have a bug report with apple, waiting to hear back from them.
This repo is just so they can reproduce the issue.
Basically, the device is not discoverable if the screen is unlocked but the app is in the background.
If you open the app or lock the device it becomes discoverable again.

**Comment by kharmabum** [comment]
@felix-dumit 

- iPhone 6s on 10.3.2 does not advertise when unlocked+background but does advertise when locked.
- iPhone 7 Plus on 10.3.2 does not advertise when unlocked+background and also does *not* advertise when the device is fully locked.

Have you noticed anything similar? Or heard back from Apple?

**Comment by felix-dumit** [comment]
I don't have a 7plus to test so I can't confirm that, but on the 6s I got the same results as you.
Haven't heard anything back from apple yet :( 
Please let me know if you figure something out
